pinot-core/src/main/java/org/apache/pinot/core/transport/grpc/GrpcResultsBlockStreamer.java

```diff
@@ -44,6 +48,8 @@ public void send(BaseResultsBlock block)
     Collection<Object[]> rows = block.getRows();
     Preconditions.checkState(dataSchema != null && rows != null, "Malformed data block");
     DataTable dataTable = block.getDataTable();
-    _streamObserver.onNext(StreamingResponseUtils.getDataResponse(dataTable));
+    Server.ServerResponse response = StreamingResponseUtils.getDataResponse(dataTable);
+    _streamObserver.onNext(response);
+    _serverMetrics.addMeteredGlobalValue(ServerMeter.GRPC_BYTES_SENT, response.getSerializedSize());
```

Comment:
Good catch on this bug. Can you also add this fix into the PR description?

Reply:
Done